samples/apply-dictionary-renderer-to-graphics-overlay/src/main/java/com/esri/arcgismaps/sample/applydictionaryrenderertographicsoverlay/components/ApplyDictionaryRendererToGraphicsOverlayViewModel.kt

```diff
@@ -0,0 +1,252 @@
+    }
+
+    init {
+        viewModelScope.launch(Dispatchers.IO) {
+            // Load the scene first
+            arcGISScene.load().onFailure { throwable ->
+                messageDialogVM.showMessageDialog(throwable)
+            }
+
+            // Create and apply dictionary renderer from a web style
+            val dictionaryRendererDeferred = viewModelScope.async(Dispatchers.IO) {
+                createMil2525dDictionaryRenderer()
+            }
+
+            // Create the point graphics in separate coroutine from a local XML file
+            val graphicsDeferred = viewModelScope.async(Dispatchers.IO) {
+                makeMessageGraphics()
+            }
+
+            val dictionaryRenderer = dictionaryRendererDeferred.await()
+            val pointGraphics = graphicsDeferred.await()
+
+            // sets the graphics in the overlay
+            graphicsOverlay.apply {
+                renderer = dictionaryRenderer
+                graphics.addAll(pointGraphics)
+            }
+
+            // Sets the camera to look a the graphics in the graphics overlay
+            val extent = graphicsOverlay.extent ?: run {
+                messageDialogVM.showMessageDialog(
+                    title = "Error",
+                    description = "The extent of the graphics overlay is null."
+                )
+                return@launch
+            }
+
+            // Set the viewpoint of the scene view using a camera
+            val camera = Camera(
+                lookAtPoint = extent.center,
+                distance = 15_000.toDouble(),
+                heading = 0.toDouble(),
+                pitch = 70.toDouble(),
+                roll = 0.toDouble()
+            )
+            sceneViewProxy.setViewpointCamera(camera)
+        }
+    }
+
+    /**
+     * Create and load a DictionarySymbolStyle from a web style and use it to create a DictionaryRenderer.
+     */
+    private suspend fun createMil2525dDictionaryRenderer(): DictionaryRenderer? {
+        val portalItem =
+            PortalItem(Portal.arcGISOnline(Portal.Connection.Anonymous), "d815f3bdf6e6452bb8fd153b654c94ca")
+
+        val dictionarySymbolStyle = DictionarySymbolStyle(portalItem = portalItem)
+
+        dictionarySymbolStyle.load().onFailure { throwable ->
+            messageDialogVM.showMessageDialog(throwable)
+            return null
+        }
+
+        // Uses the "Ordered Anchor Points" for the symbol style draw rule.
+        dictionarySymbolStyle.configurations.firstOrNull { it.name.equals("model", ignoreCase = true) }
+            ?.let { configuration -> configuration.value = "ORDERED ANCHOR POINTS" }
+
+        return DictionaryRenderer(dictionarySymbolStyle = dictionarySymbolStyle)
+    }
+
+    /**
+     * Create point graphics from a local XML file containing Mil2525d message data.
+     * The XML file is downloaded by the downloader activity.
```

Comment:
I believe `MIL-STD-2525D` is a standard format name, would be helpful to use it as mentioned in the sample: 

```suggestion
     * Create point graphics from a local XML file containing MIL-STD-2525D message data.
```